packages/components/tests/dummy/app/templates/components/modal.hbs

```diff
@@ -0,0 +1,729 @@
+        <dd>
+          <p>A string that helps the user maintain context when a modal dialog is open. (Note: this is NOT the title
+            text, but a small piece of text above the title text.)</p>
+        </dd>
+        <dt>...attributes</dt>
+        <dd>
+          <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+        </dd>
+      </dl>
+    </dd>
+    <dt>&lt;[M].Body&gt; <code>yielded component</code></dt>
+    <dd>
+      <p>It is an unstyled, generic container that yields as the main content of the modal dialog.</p>
+      <p>This container gets a scrollbar when the yielded content exceeds the available space.</p>
+      <dl class="dummy-component-props">
+        <dt>...attributes</dt>
+        <dd>
+          <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+        </dd>
+      </dl>
+    </dd>
+    <dt>&lt;[M].Footer&gt; <code>yielded component</code></dt>
+    <dd>
+      <p>It is a container that yields its content as the footer of the modal dialog.</p>
+      <p>We recommend using it exclusively for actions using the
+        <LinkTo @route="components.button-set"><code class="dummy-code">ButtonSet</code></LinkTo>
+        component. If a tertiary action is presented, it will always be aligned at the end of the row.</p>
+      <dl class="dummy-component-props">
+        <dt>...attributes</dt>
+        <dd>
+          <p><code class="dummy-code">...attributes</code> spreading is supported on this component.</p>
+        </dd>
+      </dl>
+    </dd>
+  </dl>
+</section>
+
+<section>
+  <h3 class="dummy-h3" id="how-to-use"><a href="#how-to-use" class="dummy-link-section">§</a> How to use</h3>
+
+  <h4 class="dummy-h4">Browser support</h4>
+  <p class="dummy-paragraph">
+    The
+    <code class="dummy-code">Hds::Modal</code>
+    component leverages the
+    <code class="dummy-code">&lt;dialog&gt;</code>
+    element which is currently supported by all major browser vendors. To ensure support on older browser versions (for
+    example, Safari 14 or Internet Explorer 11) we rely on
+    <a href="https://github.com/GoogleChrome/dialog-polyfill" rel="noopener noreferrer" target="_blank">a polyfill</a>
+    that is automatically loaded when needed.
+  </p>
+
+  <h4 class="dummy-h4">Page scroll</h4>
+  <p class="dummy-paragraph">
+    When an
+    <code class="dummy-code">Hds::Modal</code>
+    component is open the rest of the page becomes inert. However, scrolling at the page level is still available. To
+    make it clear to the users that the underlying elements are not interactive and to avoid confusion, we recommend
+    disabling the page scroll
+    <code class="dummy-code">onOpen</code>
+    and enabling it back
+    <code class="dummy-code">onClose</code>
+    (for example, by setting
+    <code class="dummy-code">overflow: hidden;</code>
+    and
+    <code class="dummy-code">overflow: auto;</code>
+    respectively).
```

Comment:
maybe would be better to specify that the `overflow` property should be applied to the `<body>` element of the page? (may be not obvious).

Reply:
[Updated](https://github.com/hashicorp/design-system/pull/631/commits/59d7b136cfb84fbe62d3d21d711fdcfd25b3f424) to make this more explicit.